pyseir/cli.py

```diff
@@ -165,18 +170,18 @@ def generate_state_report(state):
 
 @entry_point.command()
 @click.option('--state', default='', help='State to generate files for. If no state is given, all states are computed.')
-@click.option('--output-interval-days', default='', type=int, help='Number of days between outputs for the WebUI payload.')
-@click.option('--run-mode', default='default', type=str, help='State to generate files for. If no state is given, all states are computed.')
+@click.option('--output-interval-days', default=4, type=int, help='Number of days between outputs for the WebUI payload.')
+@click.option('--run-mode', default=DEFAULT_RUN_MODE, type=str, help='State to generate files for. If no state is given, all states are computed.')
 @click.option('--states-only', default=False, is_flag=True, type=bool, help='Only model states')
 def map_outputs(state, output_interval_days, run_mode, states_only):
     _map_outputs(state, output_interval_days=int(output_interval_days), run_mode=run_mode, states_only=states_only)
 
 
 @entry_point.command()
 @click.option('--state', default=None, help='State to generate files for. If no state is given, all states are computed.')
-@click.option('--run-mode', default='default',type=str, help='State to generate files for. If no state is given, all states are computed.')
+@click.option('--run-mode', default=DEFAULT_RUN_MODE,type=str, help='State to generate files for. If no state is given, all states are computed.')
```

Comment:
Are these an enum? anyways, to make this more explicit you can add a: 
```
type=click.Choices([run_mode.value for run_mode en RunModeEnum])
```

Reply:
Thank you!